core/src/container.rs

```diff
@@ -59,31 +95,42 @@ where
         resolved_port
     }
 
-    pub fn block_until_ready(&self) {
+    /// Returns a reference to the [`Image`] of this container.
+    ///
+    /// Access to this is useful if the [`arguments`] of the [`Image`] change how to connect to the
+    /// container (for example, because of authentication information).
+    ///
+    /// [`Image`]: trait.Image.html
+    /// [`arguments`]: trait.Image.html#associatedtype.Args
+    pub fn image(&self) -> &I {
+        &self.image
+    }
+
+    fn block_until_ready(&self) {
         debug!("Waiting for container {} to be ready", self.id);
 
         self.image.wait_until_ready(self);
 
         debug!("Container {} is now ready!", self.id);
     }
 
-    pub fn image(&self) -> &I {
-        &self.image
-    }
-
-    pub fn stop(&self) {
+    fn stop(&self) {
         debug!("Stopping docker container {}", self.id);
 
         self.docker_client.stop(&self.id)
     }
 
-    pub fn rm(&self) {
+    fn rm(&self) {
         debug!("Deleting docker container {}", self.id);
 
         self.docker_client.rm(&self.id)
     }
 }
 
+/// The destructor implementation for a Container.
+///
+/// As soon as the container goes out of scope, the destructor will either only stop or completely the docker container.
```

Comment:
Missing word: "the destructor will only stop or completely *remove the docker container"